fixes of i18n on readme.md (#329)
vtex-apps/minicart: docs/README.md

[![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors-)
<!-- ALL-CONTRIBUTORS-BADGE:END -->

> > ⚠️ **Minicart v1 block has been deprecated in favor of Minicart v2**, which can be customized using the blocks defined by [Product List](https://developers.vtex.com/docs/apps/vtex.product-list) and [Checkout Summary](https://developers.vtex.com/docs/apps/vtex.checkout-summary). If you are still using Minicart v1, see the [Minicart v1 documentation](https://github.com/vtex-apps/minicart/blob/383d7bbd3295f06d1b5854a0add561a872e1515c/docs/README.md).

Reviewer: ```suggestion
> ⚠️ Minicart v1 block has been deprecated in favor of Minicart v2, which can be customized using the blocks defined by [Product List](https://developers.vtex.com/docs/apps/vtex.product-list) and [Checkout Summary](https://developers.vtex.com/docs/apps/vtex.checkout-summary). If you still use Minicart v1, see the [Minicart v1 documentation](https://github.com/vtex-apps/minicart/blob/383d7bbd3295f06d1b5854a0add561a872e1515c/docs/README.md).
```

> The VTEX Minicart is a block that displays a summary list of all items added to the shopping cart, fetched from the Checkout OrderForm API.

Reviewer: ```suggestion
The Minicart is a block that displays a summary list of all items added to the shopping cart, fetched from the Checkout OrderForm API.
```

 },
```

> > ⚠️ **Minicart v2 will only effectively function if the store uses the** [**Add to Cart button**](https://vtex.io/docs/components/content-blocks/vtex.add-to-cart-button/) **instead of the** [**Buy button**](https://vtex.io/docs/components/content-blocks/vtex.store-components/buybutton/) in blocks such as the Shelf and the Product Details page. This is because Minicart v2 was built based on an indirect dependency on the Add to Cart button. This means that even if Minicart v2 is correctly configured in the code, it cannot be rendered by other shopping buttons, such as the Buy button.

Reviewer: ```suggestion
> ⚠️ Minicart v2 will only effectively function if the store uses the [`add-to-cart` app](https://developers.vtex.com/docs/apps/vtex.add-to-cart-button) instead of the [`buy-button` block](https://developers.vtex.com/docs/apps/vtex.store-components/buybutton) in blocks such as the Shelf and the Product Details page. This is because Minicart v2 was built based on an indirect dependency on the Add to Cart button. This means that even if Minicart v2 is correctly configured in the code, it cannot be rendered by other shopping buttons, such as the Buy button.
```
